sycl/test/basic_tests/esimd/flat_block.cpp

```diff
@@ -0,0 +1,20 @@
+// RUN: %clangxx -fsycl -fsycl-explicit-simd -fsycl-device-only -fsyntax-only -Xclang -verify %s
+// expected-no-diagnostics
+
+#include <CL/sycl.hpp>
+#include <limits>
+#include <sycl_esimd.hpp>
+#include <utility>
+
+using namespace sycl::intel::gpu;
+using namespace cl::sycl;
+
+void kernel(accessor<int, 1, access::mode::read_write, access::target::global_buffer> &buf) __attribute__((sycl_device)) {
+  simd<int, 32> v1(0, 1);
+
+  auto v0 = flat_block_load<int, 32>(buf.get_pointer());
+
+  v0 = v0 + v1;
+
+  flat_block_store<int, 32>(buf.get_pointer(), v0);
```

Comment:
Should this be `block_store` now?

Reply:
see comment above

Reply:
Renamed `flat_block_store` to `block_store`.